styles.css

```diff
@@ -0,0 +1,57 @@
+body {
+    background-image: url("https://images.unsplash.com/photo-1524169220946-12efd782aab4?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80");
+    background-size: cover;
+    background-repeat: no-repeat;
+   }
+   
+   #title {
+     color: #ffffff;
+     font-weight: bolder;
+     display: flex;
+     align-items: center;
+     justify-content: center;
+   }
+   
+   #description {
+     color: #fff;
+     align-items: center;
+     justify-content: center;
+     display: flex;
+     font-size:20px;
+   }
+   
+   #survey-form {
+     border-style: groove;
+     width: 350px;
+     height:450px;
+     line-height: 2.1;
+     padding: 20px 60px;
+     margin: 8% auto;
+     border-radius: 20px;
+     background-color: #add8e6
+   }
+   
+   #survey-form label {
+     font-weight: bolder;
+   }
+   
+   #survey-form input {
+     cursor: pointer;
+   }
+   
+   #survey-form p{
+     font-weight: bolder;
+   }
+   
+   
```

Comment:
Please don't leave empty spaces in your code. It will give linters errors.

Reply:
As I have suggested in the [previous comment](https://github.com/ngeelani48/survey-form-review/pull/1#discussion_r1071214479), I think you can be suggestive rather than instructive or giving orders, you can see reasons from the previous comments🤗.

Reply:
Comments are updated ❤️